packages/search-ui/src/SearchDriver.js

```diff
@@ -130,6 +136,15 @@ export default class SearchDriver {
       urlState = {};
     }
 
+    // Manage screen reader accessible notifications
+    this.a11yNotifications = a11yNotifications;
+    if (this.a11yNotifications) a11y.getLiveRegion();
```

Comment:
Why do you call this here? Is this simply to initialize the live region? Is it necessary to do this at initialization, or can it wait and be lazily initialized?

Reply:
It cannot be lazily initialized unfortunately, it's important to get this rendered ASAP on the page. Check out this note from [MDN's live region documentation](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions):

>Note: Assistive technologies will announce dynamic changes in the content of a live region. The live region must first be present (and usually empty), so that the browser and assistive technologies are aware of it. Any subsequent changes are then announced.
>
> ...
>
> Dynamically adding an element with an aria-live attribute or specialized role to the document also won't result in any announcement by assistive technologies (as, at that point, the browser/assistive technologies are not aware of the live region yet, so cannot monitor it for changes).
>
> Always make sure that the live region is present in the document first, and only then dynamically add/change any content.

Let me know if that makes sense to you!

Reply:
Makes sense to me, thank you for the knowledge 💣 .